### ISSUE TITLE
Make the headings on the landing page anchor links

### ISSUE THREAD

**Issue by rossbar** [numpy/numpy.org#552]
It would be nice if the top-level headings (e.g. `Ecosystem`, `Case Studies`) were anchor links so that people could link to a specific portion of the page without having to scroll to get there.

**Comment by stefanv** [comment]
They are anchor links, we should just rewrite the URL when a user clicks on sidebar items so you know *what* IDs they are :)

https://scientific-python.org/ecosystem/#install

**Comment by rossbar** [comment]
> They are anchor links, we should just rewrite the URL when a user clicks on sidebar items so you know what IDs they are :)

I'm a bit confused by this because AFAICT the numpy.org landing page doesn't have a sidebar. I was thinking something more along the lines of the little chain-link icon that appears when you hover over a section heading in some themes. Whether it's adding a sidebar or some other approach, it'd be nice to be able to easily create those anchor links to the section headings in the browser!

**Comment by stefanv** [comment]
This is addressed in https://github.com/scientific-python/scientific-python-hugo-theme/pull/116

**Comment by stefanv** [comment]
(I did not add the little chain-link icons, but it would be straightforward to do so, if desired.)

**Comment by jarrodmillman** [comment]
See #553.

**Comment by rossbar** [comment]
This is a nice improvement but it doesn't seem to apply to the landing page where there is no sidebar. AFAICT there's still no way to click on/near the `ECOSYSTEM` heading or `CASE STUDIES` headings for example.

**Comment by stefanv** [comment]
Thanks for noticing those, I will fix them too!

**Comment by tinaoberoi** [comment]
Hi everyone, 
Is this issue still open? If yes would love to work on it.

**Comment by InessaPawson** [comment]
@tinaoberoi, feel free to work on this issue.

**Comment by rossbar** [comment]
>  Is this issue still open? If yes would love to work on it.

Yes please feel free. Note though that this may be better handled in the underlying [scientific-python-hugo-theme](https://github.com/scientific-python/scientific-python-hugo-theme/)
